revoke/revoke.go

```diff
@@ -58,10 +58,13 @@ func ldapURL(url string) bool {
 //	false, false: an error was encountered while checking revocations.
 //
 //	false, true:  the certificate was checked successfully and
-//		      it is not revoked.
+//		            it is not revoked.
```

Comment:
Wyatt, can you change this one line and squash it into your commit? Once that is done, we will merge.